README.md

```diff
@@ -144,10 +144,10 @@ By default, the user is redirected to the login page within the same window, but
 ```html
 <html>
 <head>
-<script src="/path/to/solidAuthFetcher.bundle.js"></script>
+<script src="/path/to/solidClientAuthn.bundle.js"></script>
```

Comment:
This looks like it alternates between `solidClientAuthn.bundle.js` and `solid-client-authn.bundle.js` across different pages and examples. Which is accurate?

Reply:
Good catch. Is there a convention for bundle names ? Otherwise I'll go for `solid-client-authn.bundle.js`.

Reply:
Not really - I'd go with the `-`s.

Reply:
Fixed in f6f43b0e93a436a6eb0d8bd11804fefaa9af3019